contracts/contracts/strategies/CurvePoolBooster.sol

```diff
@@ -0,0 +1,334 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { Initializable } from "../utils/Initializable.sol";
+import { Strategizable } from "../governance/Strategizable.sol";
+import { ICampaignRemoteManager } from "../interfaces/ICampaignRemoteManager.sol";
+
+/// @title CurvePoolBooster
+/// @author Origin Protocol
+/// @notice Contract to manage interactions with VotemarketV2 for a dedicated Curve pool/gauge.
+contract CurvePoolBooster is Initializable, Strategizable {
+    using SafeERC20 for IERC20;
+
+    ////////////////////////////////////////////////////
+    /// --- CONSTANTS && IMMUTABLES
+    ////////////////////////////////////////////////////
+    /// @notice Base fee for the contract, 100%
+    uint16 public constant BASE_FEE = 10_000;
```

Comment:
nit: FEE_BASE might be a more organic name for this var. BASE_FEE might indicate that this is some base fee value, where some additional fees can be applied. Where FEE_BASE sounds more like BASE digits for the FEE. Feel free to disagree though :)

Reply:
Done in this commit: https://github.com/OriginProtocol/origin-dollar/pull/2327/commits/f9d4d9b2bd9d52b2fbfd6f6d660e5339b344bda6